### DISCUSSION TITLE
Falcon Discover - Create a Get-Applications cmdlet, please

### DISCUSSION THREAD

**Discussion by guy-user** [CrowdStrike/psfalcon#279] [Ideas]
Keen to be able to pull data from this part of the Falcon Discover API, when you can get to this 🙏

https://falcon.crowdstrike.com/documentation/197/falcon-discover-apis#monitor-applications

Thanks
Guy

**Comment by bk-cs** [comment]
This has already been added for the next release (v2.2.5).

**Reply by guy-user** [reply]
Awesome 😊

Any feel for a rough release date yet?

**Reply by bk-cs** [reply]
Not yet, but you can install the command yourself by copying this file over `Public\discover.ps1` in your local module directory:
```powershell
(Invoke-WebRequest https://raw.githubusercontent.com/CrowdStrike/psfalcon/2.2.5/Public/discover.ps1).content > $HOME\Documents\WindowsPowerShell\Modules\PSFalcon\2.2.4\Public\discover.ps1
```

Because there have been some other changes, you'll also need to update the private functions:
```powershell
(Invoke-WebRequest https://raw.githubusercontent.com/CrowdStrike/psfalcon/2.2.5/Private/Private.ps1).content > $HOME\Documents\WindowsPowerShell\Modules\PSFalcon\2.2.4\Private\Private.ps1
```

The new APIs are part of the `Get-FalconAsset` command and used with the `-Application` switch:
```powershell
Get-FalconAsset -Application
```

All examples were written for use with PowerShell 5.1.

**Reply by DrBob5188** [reply]
@guy-user what did you end up doing with PSFalcon for application monitoring?
